docs-kits/kits/Quality-Kit/Software Development View/page_software-development-view.md

```diff
@@ -175,9 +182,11 @@ CX_release32_fleetvehicles_100_testdata_100_parquet
 
 ### Fleet Diagnostic Sample Data
 
-Github Link to semantic data model: [CX-00038 Fleet Diagnostic Data](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.fleet.diagnostic_data)
+Github Link to semantic data
+model: [CX-00038 Fleet Diagnostic Data](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.fleet.diagnostic_data)
 
-Filedownload: <a target="_blank" href="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io.largefiles/raw/main/quality-kit/20230707-io.catenax.fleet.diagnostic_data.zip" download>20230707-io.catenax.fleet.diagnostic_data.zip</a>
+Filedownload: <a target="_blank" href="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io.largefiles/raw/main/quality-kit/20230707-io.catenax.fleet.diagnostic_data.zip" download>
```

Comment:
Not sure about HTML in Markdown. Why istnt it a normal downloadlink?

[Filedownload:](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io.largefiles/raw/main/quality-kit/20230707-io.catenax.fleet.diagnostic_data.zip)

Reply:
Did not check on content, just relied on results of linter and it did not complain. But you're right.

Reply:
Both HTML and Markdown work in Docusaurus. Since @carslen isn't responsible for the content in this PR is related to the linting only, I would leave it as it is.